core-ui/src/components/Clusters/views/ClusterList.js

```diff
@@ -163,6 +164,19 @@ function ClusterList() {
     />
   );
 
+  const loadDefaultClusterButton = (
+    <>
+      {features?.KUBECONFIG_ID?.config?.defaultKubeconfig && (
```

Comment:
We need to also check if `KUBECONFIG_ID` is enabled, because 
```json
"KUBECONFIG_ID": {
  "isEnabled": false,
  "config": {
    "kubeconfigUrl": "/kubeconfig",
    "defaultKubeconfig": "tets.yaml"
  }
},
```
will render the button, while the feature is disabled and clicking the button won't load the kubeconfig.

Reply:
done